src/app/score/winner/winner.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-winner',
+  templateUrl: './winner.component.html',
+  styleUrls: ['./winner.component.scss'],
+})
+export class WinnerComponent {
+  @Input() winner!: string;
```

Comment:
Enum? with draw option?

Reply:
I don't think this will be necessary because the visibility of the badge will be triggered by the stream from the service, and "winner" input is only to apply the correct styles to the badge. I will rename it to something less misleading.